temoa-uiuc/data_files/UIUC_2050_cost.sql

```diff
@@ -0,0 +1,3078 @@
+INSERT INTO "Demand" VALUES('uiuc',2022, 'USTM', 605.2, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2023, 'USTM', 611.2, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2024, 'USTM', 617.4, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2025, 'USTM', 623.5, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2026, 'USTM', 629.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2027, 'USTM', 636.1, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2028, 'USTM', 642.4, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2029, 'USTM', 648.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2030, 'USTM', 655.3, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2031, 'USTM', 661.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2032, 'USTM', 668.5, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2033, 'USTM', 675.2, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2034, 'USTM', 681.9, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2035, 'USTM', 688.7, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2036, 'USTM', 695.6, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2037, 'USTM', 702.6, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2038, 'USTM', 709.6, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2039, 'USTM', 716.7, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2040, 'USTM', 723.9, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2041, 'USTM', 731.1, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2042, 'USTM', 738.4, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2043, 'USTM', 745.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2044, 'USTM', 753.3, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2045, 'USTM', 760.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2046, 'USTM', 768.4, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2047, 'USTM', 776.1, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2048, 'USTM', 783.8, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2049, 'USTM', 791.7, 'thermal GWh', 'from 2015 eDNA data');
+INSERT INTO "Demand" VALUES('uiuc',2050, 'USTM', 799.6, 'thermal GWh', 'from 2015 eDNA data');
+-- Assumes 1% growth
+INSERT INTO "Demand" VALUES('uiuc',2021, 'UVCL', 129.3, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
```

Comment:
There is a large diff between this file and the current version of `11_uiuc.sql`. In the latter, the `UCVL` demand is a constant 551.5, but retains the `-- Assumes 1% growth`` comment. Which is correct @robfairh ?

Reply:
You should do either of the following:

1- Extend the following lines up to 2050:

```sql
INSERT INTO "Demand" VALUES('uiuc',2021, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2022, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2023, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2024, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2025, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2026, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2027, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2028, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2029, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2030, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
```

2-  
Extend those lines up to 2050 and assume a 1% growth.
```sql
INSERT INTO "Demand" VALUES('uiuc',2021, 'UVCL', 551.5, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2022, 'UVCL', 557.015, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
INSERT INTO "Demand" VALUES('uiuc',2023, 'UVCL', 562.58, 'K_gal of Gasoline Equiv.', 'from 2019 F&S data');
```
and so on.

The current values in ```UIUC_2050_cost.sql``` are consistent with the values in [10_uiuc.sql](https://github.com/arfc/pride/blame/master/temoa-uiuc/data_files/10_uiuc.sql#L247) but they should be consistent with the values in [11_uiuc.sql](https://github.com/arfc/pride/blame/master/temoa-uiuc/data_files/11_uiuc.sql#L247). However, the values in ```11_uiuc.sql``` are the right values but for brevity, we assumed them constant. To be consistent, you could choose the  2nd approach I described here. And update ```11_uiuc.sql``` as well. If you do such a thing, clean your ```11_uiuc.sql``` first cause it has conflicts with the file in the master branch.

Reply:
@yardasol let me know if I was clear enough.

Reply:
@yardasol Keep the constant demand. I feel like that's an appropriate assumption.